src/components/CodeExample/CodeExample.js

```diff
@@ -0,0 +1,60 @@
+import React, {Component} from 'react';
+import {colors, media} from 'theme';
+import CodeEditor from '../CodeEditor/CodeEditor';
+
+class CodeExample extends Component {
```

Comment:
Added this CodeExample component that more cleanly encapsulates the loading/CodeEditor behavior. We can remove a fair chunk of code with this!

Reply:
Ha! Fair point :) Bit of both, but we did get to remove `src/utils/mountCodeExample.js` which is nice!